### PR TITLE
Doc: rename repo ansible-avd -> avd

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Update repo URL and name from ansible-avd to avd.

This will remain a draft until change the repo name from ansible-avd to avd.
CI should be run after the repo rename to ensure `Collection code testing / Validate mkdoc content` is successful

Note: I didn't update any of the Ansible changelog fragments backups or artifacts.

### CODE REVIEW

File: python-avd/scripts/export_test_vars.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 
-# Ansible Playbook to export vars from ansible-avd molecule scenarios
+# Ansible Playbook to export vars from avd molecule scenarios
```
Comment (ClausHolbechArista) [inline python-avd/scripts/export_test_vars.yml]: AVD

**Review by ClausHolbechArista — APPROVED** [review]

File: python-avd/scripts/export_test_vars.yml (hunk 1 above)
Comment (ClausHolbechArista) [reply python-avd/scripts/export_test_vars.yml]:
```suggestion
# Ansible Playbook to export vars from AVD molecule scenarios
```

**Comment by pmprado** [comment]
Probably not very important... but this example mentions ansible-avd:

```
├── git_projects
│   ├── avd
│   ├── ansible-cvp
│   ├── ansible-avd-cloudvision-demo
│   ├── Makefile
```

This is in docs/contribution/setup-environment.html

LGTM otherwise!

**Comment by ClausHolbechArista** [comment]
@pmprado the directory you refer to is mentioning a different repo which is not being renamed here: https://github.com/arista-netdevops-community/ansible-avd-cloudvision-demo

File: ansible_collections/arista/avd/docs/contribution/setup-git.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ In this setup, the Git repository will be used by Ansible as a collection. It's
...
 
 # Move to git folder
-cd ansible-avd
+cd avd
 ```
 
 ### Update your `ansible.cfg`
 
-In your project, update your `ansible.cfg` file to point **collection_paths** to your local version of ansible-avd
+In your project, update your `ansible.cfg` file to point **collection_paths** to your local version of avd
```
Comment (ankudinov) [inline ansible_collections/arista/avd/docs/contribution/setup-git.md]: change to `AVD` or better `AVD collection`

**Review by ankudinov — COMMENTED** [review]
Search for `ansible-avd` lists around 91 files for me. Not all of the are relevant.
Nevertheless, it's feasible to this about:
- AVD examples. Currently all configs, etc. will be created in `ansible-avd-examples`. Should this be renamed or we keep it as is for now?
- There are some smaller files in `ansible_collections/arista/avd/` that are not part of the PR. Like `utils.py`

File: ansible_collections/arista/avd/docs/contribution/setup-git.md (hunk 2 above)
Comment (carlbuchmann) [inline ansible_collections/arista/avd/docs/contribution/setup-git.md]:
```suggestion
In your project, update your `ansible.cfg` file to point **collection_paths** to your local version of the `arista.avd` collection
```

**Review by ankudinov — APPROVED** [review]
LGTM

**Pull request merged**